src/main/java/org/genomenexus/vep_wrapper/VepRunner.java

```diff
@@ -40,9 +40,33 @@ public class VepRunner {
     private String vepAssembly;
 
     // Path is relative to the VEP_WORK_DIRECTORY_PATH
-    @Value("${vep.fastaFileRelativePath:homo_sapiens/98_GRCh37/Homo_sapiens.GRCh37.75.dna.primary_assembly.fa.gz}")
+    @Value("${vep.fastaFileRelativePath:homo_sapiens/112_GRCh37/Homo_sapiens.GRCh37.dna.primary_assembly.fa.gz}")
     private String vepFastaFileRelativePath;
 
+    @Value("${database.grch37.host}")
+    private String grch37DatabaseHost;
+
+    @Value("${database.grch37.port}")
+    private String grch37DatabasePort;
+
+    @Value("${database.grch37.user}")
+    private String grch37DatabaseUser;
+
+    @Value("${database.grch37.password}")
+    private String grch37DatabasePassword;
+
+    @Value("${database.grch38.host}")
+    private String grch38DatabaseHost;
+    
+    @Value("${database.grch38.port}")
+    private String grch38DatabasePort;
+
+    @Value("${database.grch38.user}")
+    private String grch38DatabaseUser;
+
+    @Value("${database.grch38.password}")
+    private String grch38DatabasePassword;
```

Comment:
let's use similar setup here as for the genome nexus backend (one vep per genome nexus instance), so maybe this can be simplified to `database.host` `database.port` etc